build_modules/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.3.0-dev
+
+- Removed the `kernelSummaryExtension`, and renamed the `KernelSummaryBuilder`
```

Comment:
can you make it more clear how this is breaking?

Reply:
How so? The only breaking things are the two items described here...

Reply:
Fair enough